README.md

```diff
@@ -160,6 +160,9 @@ import (
 )
 ```
 
+# Viewing job artifacts in openshift-ci
```

Comment:
perhaps the title could say something like "troubleshooting issues with e2e-tests in openshift-ci"?

and then add a short description of job artifacts useful for debugging as [Radim suggests](https://github.com/redhat-appstudio/e2e-tests/pull/355/files#r1108458783)?